app/views/trains/_form.html.slim

```diff
@@ -0,0 +1,28 @@
+= form_for :train, method: :patch, url: train_path, local: true do |f|
+  = render partial: "error", locals: flash
+  p
+    = f.label :name
+    br
+    = f.text_field :name, value: @train["name"]
```

Comment:
if you have a helper for `train` dont use `@train` otherwise the helper isnt doing anything